.readthedocs.yaml

```diff
@@ -0,0 +1,53 @@
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html
+version: 2
+
+build:
+  os: "ubuntu-24.04"
+  tools:
+    nodejs: "20"
+  jobs:
+    # Install micromamba
+    pre_create_environment:
+      # The recommended syntax in the micromamba docs is process substitution.
+      # RTD's default shell chokes on it, so we're doing it in multiple steps
+      # instead.
+      - curl -L micro.mamba.pm/install.sh > /tmp/micromamba-install.sh
+      - chmod +x /tmp/micromamba-install.sh
+      # For some reason, SHELL is not set. The install script requires it.
+      - SHELL=/bin/bash /bin/bash /tmp/micromamba-install.sh
+
+    # Override the create_environment step (undocumented feature) to create
+    # multiple environments.
+    create_environment:
+      # Create the env for building the docs
+      - /bin/bash --login -c "micromamba env create -n jupytergis-docs -f docs/environment-docs.yml"
+      # Create the isolated env for building JupyterGIS
+      - /bin/bash --login -c "micromamba create -n jupytergis-build -c conda-forge yarn=3 hatch pip python=3.10"
+      - /bin/bash --login -c "micromamba run -n jupytergis-build pip install -r requirements-build.txt"
+
+    # Override the install step to do nothing - we already created the envs
+    install:
+      - "echo 'Skipping! We already have the environments we need.'"
+
+    # Before building the docs, build JupyterGIS in its isolated environment,
+    # then install the wheels into the docs environment.
+    pre_build:
+      - /bin/bash --login -c "micromamba run -n jupytergis-build yarn install"
+      - /bin/bash --login -c "micromamba run -n jupytergis-build yarn dev"
+      - /bin/bash --login -c "micromamba run -n jupytergis-build jupyter labextension list 2>&1 | grep -ie 'jupytergis-core.*OK'"
+      - /bin/bash --login -c "micromamba run -n jupytergis-build jupyter labextension list 2>&1 | grep -ie 'jupytergis-lab.*OK'"
+      - /bin/bash --login -c "micromamba run -n jupytergis-build yarn build:packages"
+
+      - >-
```

Comment:
The more I look at the command below, the more I feel this isn't super readable. Most readers probably expect the line break to indicate end of the command, but the `>` folding style operator I chose (why? :shrug: I forgot) violates that intuition. I'm thinking we should switch to `|` literal style and escape the newlines to be more intuitive.

Reply:
63f605f

What do y'all think?